langchain-core/src/runnables/passthrough.ts

```diff
@@ -95,6 +95,78 @@ export class RunnableAssign<
   }
 }
 
+/**
+ * A runnable that assigns key-value pairs to inputs of type `Record<string, unknown>`.
+ */
+export class RunnablePick<
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  RunInput extends Record<string, any> = Record<string, any>,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  RunOutput extends Record<string, any> | any = Record<string, any> | any,
+  CallOptions extends RunnableConfig = RunnableConfig
+> extends Runnable<RunInput, RunOutput> {
+  lc_namespace = ["langchain_core", "runnables"];
+
+  keys: string | string[];
+
+  constructor(keys: string | string[]) {
+    super();
```

Comment:
Serialization?

Reply:
Okay this was broken in RunnableAssign too, fixed both now